### DISCUSSION TITLE
Write to file with sudo/doas

### DISCUSSION THREAD

**Discussion by BatteredBunny** [helix-editor/helix#10214] [General]
Would it be possible to get a feature for attempting to write to file with sudo/doas when not enough permissions? 
[Similar to how micro does it](https://github.com/zyedidia/micro/issues/158)
https://github.com/zyedidia/micro/blob/master/runtime/help/options.md#sucmd

**Comment by kirawi** [comment]
Duplicate of https://github.com/helix-editor/helix/discussions/4251

**Comment by pchampin** [comment]
I respectfully disagree that this is a duplicate of #4251. That discussion is about *starting* helix with sudo.

The question in this issue, and the linked-to feature of micro, is about *writing* a file with sudo, in an instance of helix that was started *as a regular user*. I agree that this would be a super useful feature.

**Comment by kirawi** [comment]
They are actually duplicates because the question in the discussion was not about starting Helix with sudo, but generally how to write with sudo. There are mentions of the system from micro as well. I'll reopen since those mentions weren't addressed in that discussion though.

**Comment by the-mikedavis** [comment]
I don't want to see this added to core. It's a hack as micro says that pipes the buffer contents through `sudo tee` and you can do that already with the shell_pipe command: `%|sudo tee <path>`, using `C-z` to suspend and `sudo -v && fg` in the terminal so you don't have to enter the password.
